packages/web_mouse_click_boundary/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## [0.0.1] - TODO: Add release date.
```

Comment:
TODO: Edit this, and ensure that the version of the package in pubspec is correct.